sdk/src/manifest_store_report.rs

```diff
@@ -105,8 +108,18 @@ impl ManifestStoreReport {
         let cert_str = store.get_provenance_cert_chain()?;
         println!("{cert_str}\n\n");
 
-        if let Some(ocsp_info) = store.get_ocsp_status() {
-            println!("{ocsp_info}");
+        let claim_label = store
+            .provenance_path()
+            .ok_or(crate::Error::ProvenanceMissing)?;
+        if let Some(ocsp_info) = store.get_ocsp_status(&claim_label) {
+            if let Some(revoked_at) = &ocsp_info.revoked_at {
+                println!("Certificate Status: Revoked, revoked at: {}", revoked_at);
```

Comment:
If there is a update time and it has expired without error then may we should just display a "Not Revoked" or "Cert Good" message.